neo/IO/Json/JString.cs

```diff
@@ -13,9 +13,7 @@ public class JString : JObject
 
         public JString(string value)
         {
-            if (value == null)
-                throw new ArgumentNullException();
-            this.Value = value;
+            this.Value = value ?? throw new ArgumentNullException();
```

Comment:
Time to change to "newtonsoft json" ?

Reply:
Newton, the apple in the head guy? AHeuahuea
Are there much efficiency difference, Shargon?

Reply:
Newtonsoft is great for easy implementation and overall support, but I think it's not the best choice for performance. See https://github.com/neuecc/Utf8Json 

![30883721-11e0526e-a348-11e7-86f8-efff85a9afe0](https://user-images.githubusercontent.com/17299848/41978922-b609720e-7a1a-11e8-87ea-e58f942c36bf.png)

Reply:
Hey, Bruno.
Thanks for these findings. Great insights and in an important time.

Where are you from? Sounds Brazilian name.

Reply:
newtonsoft is the best and more tested library of Json in .net

Reply:
In this chart not appear Newtonsoft ...

Reply:
Newtonsoft is Json.net @shargon

Reply:
Bruno, and which is the native one that is being used in this refactoring?

Reply:
It seems erik kept the same strategy than before, not using external libraries. neo/IO/Json is basically a implementation of Serializing and Deserializing JSON objects. I'm ok with continuing with this route if there are no advantages in using external libraries (perfomance and code expansion wise).

Reply:
Gud gud muito bem, Bruno.
Thanks so much for this insights and positives conclusions!
Valeuuu